whitelist/whitelist.go

```diff
@@ -0,0 +1,73 @@
+// Package whitelist provides methods for managing an app's whitelisted IP's.
+package whitelist
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	deis "github.com/deis/controller-sdk-go"
+	"github.com/deis/controller-sdk-go/api"
+)
+
+// List IP's whitelisted for an app.
+func List(c *deis.Client, appID string) (api.Whitelist, error) {
```

Comment:
i thought about using the appsettings list but then i felt it is better to have them separately since whitelist uses its own endpoint and it will be consistent across list/add/delete.
But i am ok with using the appsettings one also.